tests/ui/consts/control-flow/try.rs

```diff
@@ -1,11 +1,11 @@
-// The `?` operator is still not const-evaluatable because it calls `From::from` on the error
-// variant.
+//@ run-pass
```

Comment:
This should not be `run-pass`, it doesn't have anything in its main function. It could be `check-pass`, but you'll still need more feature gates to make it compile